org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/component/Messagebox.java

```diff
@@ -55,6 +57,8 @@
 	 * 
 	 */
 	private static final long serialVersionUID = 8928526331932742124L;
+	
+	private static final CLogger log = CLogger.getCLogger(Messagebox.class);
```

Comment:
This line is creating an eclipse warning - if not used is better to remove it

Reply:
That was left over from debugging - will be removed